### DISCUSSION TITLE
Why to use this instead of Notion ?

### DISCUSSION THREAD

**Discussion by RayyanNafees** [undb-xyz/undb#1258] [Q&A]
There are many tools that provide the same functionality like Notion and also includes there API to integrate it with other applications

What makes this Product stands out when [Notion](https://notion.so) is a popular choice and also has an AI prompt

**Comment by nichenqin** [comment]
Good question, will answer it later

**Comment by ghost** [comment]
> There are many tools that provide the same functionality like Notion and also includes there API to integrate it with other applications

Private first, unified, self-hosted no code database

> What makes this Product stands out when [Notion](https://notion.so/) is a popular choice and also has an AI prompt

It is a good question. And maybe If there is any integration with [janhq/jan](https://github.com/janhq/jan) it would be interesting to be viable as an alternative to Notion(offline-first solutions for AI that improve your privacy)
